projects/ad9081/src/app_parameters.h

```diff
@@ -44,28 +44,48 @@
 /******************************************************************************/
 /********************** Macros and Constants Definitions **********************/
 /******************************************************************************/
-#define PHY_CS	0
+#ifdef XPS_BOARD_ZCU102
+#define GPIO_OFFSET		78
+#else
+#define GPIO_OFFSET		0
+#endif
+
+#define PHY_CS			0
 
 #ifdef QUAD_MXFE
-#define ADF4371_CS	0
-#define HMC7043_CS	4
+#define ADF4371_CS		0
+#define HMC7043_CS		4
 
-#define PHY_RESET		41
+#define PHY_RESET		GPIO_OFFSET + 41
```

Comment:
```suggestion
#define PHY_RESET		(GPIO_OFFSET + 41)
```

Reply:
same below

Reply:
yeah, not really needed here but I'll do it anyway, it looks better